Elmah.sln

```diff
@@ -107,4 +108,7 @@ Global
 	GlobalSection(SolutionProperties) = preSolution
 		HideSolutionNode = FALSE
 	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
```

Comment:
This is an unrelated change. Can we get rid of it?

Reply:
Removed